### PR TITLE
Announcement of 0.9.3 release

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: blog/2019-03-26-debezium-0-9-3-final-released.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
+= Debezium 0.9.3.Final Released
+jpechane
+:awestruct-tags: [ releases, mysql, mongodb, postgres, sqlserver, docker ]
+:awestruct-layout: blog-post
+
+The Debezium team is happy to announce the release of Debezium *0.9.3.Final*!
+
+This is mostly a bug-fix release and a drop-in replacement for earlier Debezium 0.9.x versions but there are few signifacnt new features too.
```
Comment (renatomefi) [inline blog/2019-03-26-debezium-0-9-3-final-released.adoc]:
```suggestion
This is mostly a bug-fix release and a drop-in replacement for earlier Debezium 0.9.x versions but there are few significant new features too.
```

File: blog/2019-03-26-debezium-0-9-3-final-released.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+[NOTE]
+Docker images will be released with a delay due to Docker Hub configuration issues.
+
+== New Features
+
+The 0.9.3 release also comes with two larger new features:
+
+* A feature request was made to execute a partial recovery of replication process from lost slot in PostgreSQL database.
+The result is a first service provider interface in Debezium that allows the user to inject itw own Java class that controls snaphotting process (https://issues.jboss.org/browse/DBZ-1082[DBZ-1082]).
```
Comment (renatomefi) [inline blog/2019-03-26-debezium-0-9-3-final-released.adoc]:
```suggestion
The result is a first service provider interface in Debezium that allows the user to inject its own Java class that controls snaphotting process (https://issues.jboss.org/browse/DBZ-1082[DBZ-1082]).
```

File: blog/2019-03-26-debezium-0-9-3-final-released.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* A regression that introduced deadlock in snapshotting process has been fixed (https://issues.jboss.org/browse/DBZ-1161[DBZ-1161])
+* `hstore` datatype works correctly in snapshot phase (https://issues.jboss.org/browse/DBZ-1162[DBZ-1162])
+* `wal2json` plug-in processes also empty events (https://issues.jboss.org/browse/DBZ-1181[DBZ-1181])
+* Commit time is propely converted to microseconds (https://issues.jboss.org/browse/DBZ-1174[DBZ-1174])
+
+
+Also the link:/docs/connectors/mysql/[Debezium MySQL connector] saw a number of fixes especially in SQL parser:
+
+* `SERIAL` datatype and default value is now supported (https://issues.jboss.org/browse/DBZ-1185[DBZ-1185])
+* A specific deatil in MySQL grammar that allowed to enumerate table options in `ALTER TABLE` without comma works (https://issues.jboss.org/browse/DBZ-1186[DBZ-1186])
```
Comment (renatomefi) [inline blog/2019-03-26-debezium-0-9-3-final-released.adoc]:
```suggestion
* A specific detail in MySQL grammar that allowed to enumerate table options in `ALTER TABLE` without comma works (https://issues.jboss.org/browse/DBZ-1186[DBZ-1186])
```

**Comment by gunnarmorling** [comment]
Hey @renatomefi, would you be interested pehaps in writing a quick blog
post in your new SMT? Showing how to use it etc. It could largely be taken
from the ref docs (which we'll need anyways), plus some nice framing of the
background etc. WDYT?

**Comment by gunnarmorling** [comment]
Amended and applied. Thanks, @jpechane!

Still curious about your thoughts on the outbox SMT blog post idea, @renatomefi. Plus the description of the SMT in the docs :) Let us know if you need help with the latter.

**Comment by renatomefi** [comment]
@gunnarmorling sure thing! I'll create a short one then!
Also as you suggested starting with fixing the example https://github.com/debezium/debezium-examples/pull/60

**Pull request merged**